charts/tornjak/values.yaml

```diff
@@ -12,17 +12,15 @@ clustername: spire-example
 
 # SPIRE related elements
 # trustdomain is arbitrary but needs to match between Server and Agent
-trustdomain: spiretest.com
+trustdomain: example.org
 # SPIRE version used for consistency across components
-spireVersion: 1.0.2
+spireVersion: 1.1.5 # OIDC only works with SPIRE 1.1.x
+# spireVersion: 1.3.5
 
 # SPIRE Server configuration
 spireServer:
   # tornjakImage - Tornjak with SPIRE Server
-  # TODO this is just a temporary image with several patches. It
-  # should be removed after the patches are available in the SPIRE main
-  # img: ghcr.io/spiffe/tornjak-spire-server
-  img: tsidentity/local-spire-server
+  img: ghcr.io/spiffe/tornjak-spire-server
```

Comment:
we will soon need to change this image name to `ghcr.io/spiffe/spire-server-tornjak-be` once the latest tornjak prs come out

Reply:
We should change the name once the new image is available. Separate PR.